Developer-Guide.md

```diff
@@ -24,6 +24,15 @@
 * [Troubleshoot Kata Containers](#troubleshoot-kata-containers)
 * [Appendices](#appendices)
     * [Checking Docker default runtime](#checking-docker-default-runtime)
+    * [Setting up a debug console](#setting-up-a-debug-console)
```

Comment:
Suggest changing this title to "Set up a debug console" to avoid using the gerund (ing) format.